offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/persons/CustomerDbConnector.groovy

```diff
@@ -232,6 +232,8 @@ class CustomerDbConnector implements CreateCustomerDataSource, UpdateCustomerDat
     statement.setString(2, customer.lastName)
     statement.setString(3, customer.title.value)
     statement.setString(4, customer.emailAddress )
+    //a new customer is always active
+    statement.setBoolean(5, true)
```

Comment:
why is it here true and not 1?

Reply:
Boolean gets converted to tinyint in our database, I was not sure if setting 1 would work, since preparedStatements provide the "setBoolean" method.